src/rgw/rgw_common.h

```diff
@@ -762,6 +763,7 @@ struct rgw_bucket {
     ::encode(index_pool, bl);
     ::encode(data_extra_pool, bl);
     ::encode(tenant, bl);
+    ::encode(name_space, bl);
```

Comment:
ok, for this one I was curious that the version number was bumped to 9 and there was no additional field in the structure, I dug a little and saw that the version number was bumped to 9 by mistake by one of my auth rework PRs !. So that is why I kept the version at 9 for 'name_space'.